http://bugs.gentoo.org/246748

This patch can be removed for next release as per:

http://bugzilla.gnome.org/show_bug.cgi?id=610666#c2

--- python/Makefile.am
+++ python/Makefile.am
@@ -36,7 +36,7 @@
 python_LTLIBRARIES = libxsltmod.la
 
 libxsltmod_la_SOURCES = libxslt.c types.c libxslt-py.c
-libxsltmod_la_LIBADD = $(mylibs) 
+libxsltmod_la_LIBADD = $(mylibs)  -lpython$(PYTHON_VERSION)
 
 libxslt.py: $(srcdir)/libxsl.py $(srcdir)/libxsltclass.py
 	cat $(srcdir)/libxsl.py $(srcdir)/libxsltclass.py > libxslt.py
